src/main/java/uk/gov/dvsa/model/cvs/certificateData/AdditionalDefect.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.dvsa.model.cvs.certificateData;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is the ignore unknown flag needed here; are we expecting any additional properties to pop up?